sip-tbd/sip-tbd-taproot_payments.md

```diff
@@ -0,0 +1,75 @@
+# Preamble
+
+SIP Number: [TBD]
+
+Title: Taproot Payments Integration
+
+Authors: Rena Shah ([rena@trustmachines.co](mailto:rena@trustmachines.co))
+
+Consideration: Governance, Technical
+
+Type: Consensus
+
+Status: Draft Proposal
+
+Created: 24 May 2022
+
+License: CC0-1.0
+
+Sign-off: Technical Committee
```

Comment:
No one has signed off on this yet, so this section should be blank

Reply:
I'll update, thanks!